tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -18,7 +18,7 @@ class RingTone(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == self.manifest_url)
```

Comment:
you can use self.wait_to_be_displayed() here. But the app needs a  manifest_url property.